src/settings/TargetProfileDetail.js

```diff
@@ -133,10 +47,10 @@ class TargetProfileDetail extends React.Component {
       id: jobProfile?.id,
       name: jobProfile?.name,
     }));
-  }
+  };
 
-  getJobProfilesFormatter = defaultProfileId => ({
-    [JOB_PROFILES_COLUMNS_NAME.ID]: ({ name, id }) => (
+  const getJobProfilesFormatter = defaultProfileId => ({
+    [JOB_PROFILES_COLUMNS_NAME.ID]: ({ name, id }) => ( // eslint-disable-line react/prop-types
```

Comment:
Lint is telling you, "Hey, this is actually a functional component and components need prop-types" so instead of shutting it off, just follow its (somewhat oblique) advice: lift this function up and out of the body of `TargetProfileDetail`, or maybe even into its own file, and treat it like the real component that it is.

Reply:
Let's leave it as is, but remove destructuring like [here](https://github.com/folio-org/ui-inventory/blob/master/src/Instance/InstanceDetails/InstanceTitleData/TitleSeriesStatements.js#L27) and [here](https://github.com/folio-org/ui-inventory/blob/ded70e7a371366947b4909706220c6fc1afae32c/src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js#L53)